.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,6 +1,7 @@
 name: Bug report 🐛
 description: Report a bug
 labels: ["category.bug", "status.needs-triage"]
+title: "[BUG] "
```

Comment:
can we improve this line a bit maybe as a call to action to succinctly describe the bug? similar to the project submission title

Reply:
maybe "Fix ___"?

Reply:
do we really need the title you think? https://github.com/paradigmxyz/reth/blob/main/.github/ISSUE_TEMPLATE/bug.yml?plain=1#L1 <- i tried out the reth "create a bug" report and thought it was nice. just an empty field to clearly fill out the title.

Reply:
Makes sense, we have the category.bug tag already